.changeset/young-llamas-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+"@clerk/nextjs": patch
+---
+
+Excluding test declartion files from module builds
```

Comment:
We can probably make this empty as there's no practical impact, and it'll get picked up on the next release.

```suggestion
---
---
```